File: MockingPeople/Models/ContactDevice.cs
namespace MockingPeopleLibrary.Models
{
    public class ContactDevice
    {
        public int Id { get; set; }
        public DeviceType DeviceType { get; set; }
        public int DeviceTypeId { get; set; }
        public string Value { get; set; }
        public override string ToString() => $"{DeviceType.Description} : {Value}";

        public ContactDevice(int id)
        {
            Id = id;
        }

        public ContactDevice()
        {
            
        }
    }
}

File: BindingNavigatorVBWinForms/Models/ContactDevice.vb


Namespace Models
	Public Class ContactDevice
		Public Property Id() As Integer
		Public Property DeviceType() As DeviceType
		Public Property DeviceTypeId() As Integer
		Public Property Value() As String
		Public Overrides Function ToString() As String
			Return $"{DeviceType.Description} : {Value}"
		End Function

		Public Sub New(ByVal id As Integer)
			Me.Id = id
		End Sub

		Public Sub New()

		End Sub
	End Class
End Namespace